lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -91,8 +95,8 @@
       <li class="action action-share">
         <a class="action-linkedin-profile" target="_blank" href="${cert_status['linked_in_url']}"
          title="${_('Add Certificate to LinkedIn Profile')}"
-         data-course-id="${unicode(course_overview.id)}"
-         data-certificate-mode="${cert_status['mode']}"
+         data-course-id="${course_overview.id | n, unicode}"
```

Comment:
This should be:
```
data-course-id="${course_overview.id}
```
The original was fine, just redundantly calling unicode which is part of the default.  The updated one was not fine, because the ``n`` would drop all defaults, including HTML-escaping, and that is the very thing we are adding in.